Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -107,6 +107,8 @@ public class SchemeGenerator {
                 }
                 pbxProj = try getPBXProj(from: projectReference)
                 projectFilePath = projectReference.path
+            case .package:
+                fatalError("Currently unsupported to use package for: \(target)")
```

Comment:
I think fatalError is not good for user to tell the error. You may can use throw error at here. Throwed error is shows as proper format as XcodeGen output Error.

Reply:
Thanks for pointing that out! 

I'm actually not sure why that isn't supported, I wonder if I did a mistake when updating this.

I think I can add it by updating it to:

```
case .package(let packageName):
    guard let package = self.project.getPackage(packageName),
        case .local(let path) = package else {
        throw SchemeGenerationError.missingPackage(packageName)
    }
    return XCScheme.BuildableReference(
        referencedContainer: "container:\(path)",
        blueprintIdentifier: target.name,
        buildableName: target.name,
        blueprintName: target.name
    )
```

Reply:
I added my suggestion, because it's probably easier to review this way.